3-UsingAPIs/Week2/assignment/ex4-diceRace.js

```diff
@@ -15,16 +15,22 @@ const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
   const dice = [1, 2, 3, 4, 5];
-  // TODO complete this function; use Promise.race() and rollDie()
+  
+ 
+  return Promise.race(
+    dice.map(() => rollDie())
+  );
```

Comment:
Convince yourself that this even shorter version works too.

```suggestion
  return Promise.race(dice.map(rollDie));
```